webauthn-for-external-idp-users/backend/routes/index.js

```
@@ -0,0 +1,79 @@
+import { randomUUID } from 'crypto';
+import express from 'express';
+import fetch from 'node-fetch';
+import { common } from '@ciam-expressjs-vanilla-samples/shared';
+
+const router = express.Router();
+let clientCredsToken;
+
+router.get(['/'], async function (req, res) {
+  res.redirect('/pages/authenticate.html');
+});
+
+router.post('/authenticate/complete', async function (req, res) {
+  try {
+    const webauthnEncodedResult = req.body.webauthn_encoded_result;
+    const url = common.config.apis.webauthnAuthenticateCompleteExternal;
+    const token = await getClientCredsToken();
+    const request = {
+      method: 'POST',
+      headers: {
+        Authorization: `Bearer ${token}`,
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        webauthn_encoded_result: webauthnEncodedResult,
+      }),
+    };
+
+    const data = await fetch(url, request);
+    const json = await data.json();
+    console.log('Complete response', json);
+
+    res.status(data.status).send(json);
+  } catch (e) {
+    console.log(e);
```

We should return an error here so that the UI will reflect it.
For example:
https://github.com/TransmitSecurity/ciam-expressjs-vanilla-samples/blob/main/webauthn-cross-device/backend/desktop.js#L14